<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPIDialog</class>
 <widget class="QDialog" name="SPIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPI connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupSpeed">
       <property name="title">
        <string>Speed</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string notr="true">30 KHz</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="Index" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string notr="true">125 KHz</string>
          </property>
          <property name="Index" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string notr="true">250 KHz</string>
          </property>
          <property name="Index" stdset="0">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string notr="true">1 MHz</string>
          </property>
          <property name="Index" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupPolarity">
         <property name="title">
          <string>Receive Polarity</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QRadioButton" name="radioButton_18">
            <property name="text">
             <string>Idle Low</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_17">
            <property name="text">
             <string>Idle High</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupEdge">
         <property name="title">
          <string>Output Clock Edge</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton_9">
            <property name="text">
             <string>Idle to active</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_10">
            <property name="text">
             <string>Active to idle</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupPhase">
         <property name="title">
          <string>Input Sample Phase</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton_13">
            <property name="text">
             <string>Middle</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_14">
            <property name="text">
             <string>End</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupOutput">
         <property name="title">
          <string>Output type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="radioButton_16">
            <property name="text">
             <string>Open drain (H=Hi-Z, L=GND)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_15">
            <property name="text">
             <string>Normal (H=3.3V, L=GND)</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SPIDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SPIDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
